trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require './decorator'
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    @nameable.correct_name[0...10]
```

Comment:
- [ ] Here we have to trim only when the length is greater than 10.Kindly use 
`@nameable.correct_name.length > 10 ? @nameable.correct_name[0..9] : @nameable.correct_name`